frappe/public/js/frappe/form/sidebar/user_image.js

```diff
@@ -73,6 +73,8 @@ frappe.ui.form.setup_user_image_event = function (frm) {
 					field.make_input();
 				}
 				field.$input.trigger("attach_doc_image");
+				// close sidebar
+				frm.refresh();
```

Comment:
Refreshing form just to close the sidebar is not a good solution

Reply:
You can do `frm.page.close_sidebar()`

For this, to work you need to convert the `close_sidebar` function from a local variable to a global `this.close_sidebar`.

Reply:
Thanks @shariquerik 
Done changes, Please check